flutter/ios/sentry_flutter.podspec

```diff
@@ -6,7 +6,7 @@ Pod::Spec.new do |s|
 Sentry SDK for Flutter with support to native through sentry-cocoa.
                        DESC
   s.homepage         = 'https://sentry.io'
-  s.license          = { :type => 'MIT', :file => '../../LICENSE' }
+  s.license          = { :type => 'MIT', :file => '../LICENSE' }
```

Comment:
the change makes sense, but `../../LICENSE` also exists, what was the warning?

Reply:
If you run `pod install` in `flutter/example/ios` this warning happens: 
```
[!] A license was specified in podspec `sentry_flutter` but the file does not exist - /Users/jonas/Documents/projects/sentry-dart/flutter/example/ios/.symlinks/plugins/LICENSE

[!] Unable to read the license file `../../LICENSE` for the spec `sentry_flutter (0.0.1)`

[!] Unable to read the license file `../../LICENSE` for the spec `sentry_flutter (0.0.1)`

[!] CocoaPods did not set the base configuration of your project because your project already has a custom config set. In order for CocoaPods integration to work at all, please either set the base configurations of the target `Runner` to `Target Support Files/Pods-Runner/Pods-Runner.profile.xcconfig` or include the `Target Support Files/Pods-Runner/Pods-Runner.profile.xcconfig` in your build configuration (`Flutter/Release.xcconfig`).

[!] Your project does not explicitly specify the CocoaPods master specs repo. Since CDN is now used as the default, you may safely remove it from your repos directory via `pod repo remove master`. To suppress this warning please add `warn_for_unused_master_specs_repo => false` to your Podfile.
```

Reply:
I see, looks like that this could depend on where your App is placed (in our case, flutter/example/ios).
wondering if a fresh project with our SDK also throws this warning or not.

Reply:
Yep, it does.